examples/basics/data_row_metadata.ipynb

```diff
@@ -3,7 +3,9 @@
   {
```

Comment:
<p>Unsure if we want to expose all of custommetadatatime as it does take up a lot of space</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Got it. Will only print out metadata info</p>